### ISSUE TITLE
Please consider adding RUnit to Suggests

### ISSUE THREAD

**Issue by HenrikBengtsson** [Bioconductor/DelayedArray#24]
`DelayedArray:::.test()` calls `BiocGenerics:::testPackage()` which calls `library("RUnit", quietly = TRUE)`.  I Since RUnit is only a suggested package of BiocGenerics, I think this causes `install.packages()` on DelayedArray to fail picking up RUnit.  This, in turn, causes automated reverse package dependency checks on DelayedArray to fail with:
```
     ERROR
    Running the tests in ‘tests/run_unitTests.R’ failed.
    Last 13 lines of output:
      
      The following objects are masked from 'package:matrixStats':
      
          colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
      
      The following objects are masked from 'package:base':
      
          aperm, apply
      
      [1] TRUE
      > DelayedArray:::.test()
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> <Anonymous> -> library
      Execution halted
```

I observe this when running [revdep checks on matrixStats](https://github.com/HenrikBengtsson/matrixStats/blob/develop/revdep/problems.md#delayedarray).  Adding RUnit to Suggests should fix this.

**Comment by hpages** [comment]
Thanks @HenrikBengtsson . This is corrected in DelayedArray [0.7.21](https://github.com/Bioconductor/DelayedArray/commit/a39ce365ea73c3187dadf1e273fee7190edd38da) (BioC devel) and 0.6.2 (BioC release).

**Comment by HenrikBengtsson** [comment]
Confirming that those are live now. Thanks.
